webapp/locales/de.json

```diff
@@ -91,9 +91,9 @@
     }
   },
   "admin": {
-    "name": "Systemverwaltung",
+    "name": "Admin",
     "dashboard": {
-      "name": "Startzentrale",
+      "name": "Dashboard",
```

Comment:
Yes I think most Germans understand the meaning of "Dashboard" by now. Literally translated it means "Armaturenbrett".

Reply:
certain words just don't work in other languages. we are forced to find our own terms. 
in the admin area it has more to do with people who can use the word dashboard. and expect something. 
linguistics is definitely an important branch and our cultural burden in europe :) 
i think it's important that we develop our own style. one world .. you can think if people are one step ahead.